artifactory/utils/golang/project/project.go

```diff
@@ -324,11 +323,10 @@ func parseModuleName(modContent string) (string, error) {
 }
 
 // Returns a regex that match the following:
-// 1. .git folder.
-// 2. .gitignore file
-// 3. .DS_Store
+// 1. .DS_Store.
+// 2. .git.
 func getPathExclusionRegExp() (*regexp.Regexp, error) {
```

Comment:
Should we also exclude files that are marked as ignored in the **.gitignore**?

Reply:
Fixed, Ignore files according to 'gitignore' pattern